src/ovp_pipeline/static/ovp-chat-drawer.css

```
@@ -0,0 +1,211 @@
+}
+
+.chat-drawer-turn-user {
+  background: var(--chat-drawer-muted);
+}
+
+.chat-drawer-body {
+  font-family: inherit;
+  font-size: 0.9375rem;
+  line-height: 1.45;
+  white-space: pre-wrap;
+  word-wrap: break-word;
```

Stylelint (17.11.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)